templates/ConfluenceDataCenter.template

```diff
@@ -206,7 +206,8 @@ Parameters:
   ConfluenceVersion:
     AllowedPattern: "(\\d+\\.\\d+\\.\\d+(-?.*))"
     AllowedValues:
```

Comment:
we are thinking to remove AllowedValues so we don't need to update CF when we add new versions. how do you think Tony ?